src/cmd/metadata.rs

```diff
@@ -34,6 +36,7 @@ const METADATA_FILE: &str = "metadata.json";
 struct GenerateMetadataCommand {
     crate_metadata: CrateMetadata,
     verbosity: Option<Verbosity>,
+    include_wasm: bool,
```

Comment:
We should consider producing a different file name, perhaps `<contract_name>.pack` or such as @Robbepop  suggested in #96.

Reply:
After a run with `cargo contract pack` we expect as output 3 files: the `.wasm`, the `metadata.json` and the `.pack` file.